scripts/deploy-to-app-center.sh

```diff
@@ -1,4 +1,4 @@
-#!/usr/bin/env bash
+#!/bin/sh
```

Comment:
I think this isn't correct - `#!/bin/sh` is used for system boot scripts, isn't it?

When you create a new script in the Bitrise UI, it automatically uses `#!/usr/bin/env bash` for Bash scripts, so I think we should keep that.

Reply:
you're right, this was a failed attempt to fix something wrong